### ISSUE TITLE
Determine whether to use TDLinx or ADD entry data

### ISSUE THREAD

**Issue by huntallcott** [huntallcott/FoodDeserts#5]
@diamondr @handbury @mollyschnell @moosedube @rakhkovs ,
The goal of this issue is to determine whether to use the TDLinx or ADD store entry data for the store entry event studies. 

As a reminder of what ADD did: Nielsen wouldn't allow us to disclose the retailer names in the paper, but ADD use entries from four supermarkets/supercenters: Walmart Supercenter (retailer_code 6920), Target Super (6921), Walmart Neighborhood (848), and Safeway (32; note that this includes only a smaller number of openings in 2005 and 2006). For the Walmart Supercenters, we started with Walmart's public list and then coded Walmart -> Supercenter conversions using data from other papers and a search of press releases. We have exact addresses for all. So although these are a limited set of retailers, we're very confident that these are grocery stores with produce sections, and we are very confident in the exact open dates and addresses.

I know that @handbury has looked into the TDLinx data extensively, but I thought I'd do one more check against the ADD entry data. I used the stata do file [here](https://github.com/huntallcott/FoodDeserts/blob/master/Code/Temp/TDLinx%20versus%20ADD%20entry%20data.do) to compare the open dates, lat/long coordinates, and store counts of stores in the TDLinx vs. ADD data. This creates output [here](https://github.com/huntallcott/FoodDeserts/tree/master/Output/Temp/TDLinx%20validation). 
For each of the four retailer codes, you can see the difference in open dates and distance in miles between pairs of stores in the same zip code that appear in both the ADD and TDLinx data.
* >85% of TDLinx store open dates are within 50 days of the ADD date
* >80% of TDLinx locations are within 0.3 miles of the ADD address
* ADD tends to comprise about 45-52% of the store openings in the combined store opening data. (If both datasets were perfect, then it would be 50-50%.) I would worry if ADD share were much lower than 50%, as this could represent phantom store openings in TDLinx. For Walmart Supercenter (retailer 6920), the ADD coverage trails off below 40% in 2012 and later. I did a few spot checks, and these look like true Supercenter conversions (original Walmarts converted to Supercenters) that happened without press releases, and thus ADD did not catch them because we only catch a Supercenter conversion when there is a press release.

Thus, by this set of measures, the TDLinx data are again appearing to be reliable for our purposes. Obviously the TDLinx data have the significant advantage of offering coverage of all retail banners.

Thoughts or comments? Unless I'm missing something, my sense is that we should throw out the ADD data and rely fully on TDLinx.

One question for @handbury or others: I notice that Walmart is only in the TDLinx data for 2015 and 2016. What chains are supposed to be in the data for which years? Lack of historical coverage could be a problem for trying to establish whether a previous store closed in the same area. I know you looked into this and am just wanting to get up to speed.

**Comment by rakhkovs** [comment]
Hi Hunt,

I checked this issue. Walmart should be in TDLinx prior to 2015. I think
the issue is that prior to 2015 it was spelled as "Wal Mart"  and from 2015
it became "Walmart". Could you please check if you can find more stores
with alternative spelling.

Ilya

**Comment by huntallcott** [comment]
Got it. Just for my understanding, what chains are supposed to be in the data for which years? Is this, for example, *all* retail establishments in the lower 48 states from the channels listed in column F/G of C:\Users\Hunt\Dropbox\FoodDeserts\Externals\Data\StoreEntryExit\TDLinx_storelist_082616.csv?

**Comment by rakhkovs** [comment]
Hunt,

This is a difficult question. We have two datasets of store "universe"
TDLinx and STARS (SNAP authorized stores). When we tried to merge the two
the overlap was about 90%, so we treated both as a universe measure. We are
not sure which chains are not in TDLinx. I suspect the gaps in coverage
will be in small/independent retailers, but without a better measurement of
small/independent stores we are not sure.

Next, about the years. In our most recent version of TDLinx, in the first
year of our data (2004) we observe 246K stores, in the last year (2017) we
have 288K. Given the difference, I don't expect there to be a lot of stores
missing in earlier years. We almost have not documentation on TDLinx and we
don't know the details of their data collection.

Ilya
